src/Turbo/README.md

```diff
@@ -260,7 +261,7 @@ class ChatController extends AbstractController
 
             // 🔥 The magic happens here! 🔥
             // The HTML update is pushed to the client using Mercure
-            // The Mercure topic (here "chat") MUST be the ID of the block that will display the received updates
+            // The Mercure topic (here "chat") MUST be the ID of the block that will display the received updates // in the twig below; we never see any id="chat", this might be confusing
```

Comment:
This comment isn't accurate anymore, calling `turbo_stream_listen('chat')` is now enough. I'll update.